web/src/main/java/by/ilyin/web/util/validator/UserValidator.java

```diff
@@ -0,0 +1,27 @@
+package by.ilyin.web.util.validator;
+
+import by.ilyin.web.exception.http.client.IncorrectValueFormatException;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.ObjectError;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class UserValidator {
+
+    public void userValidationProcess(BindingResult bindingResult) {
```

Comment:
@YauhenIlyin What is the purpose of this method?
Did you investigate how BindingResult works?
Why Validator interface is missing?